src/native.js

```diff
@@ -154,22 +154,36 @@ export function fireNativeTrackers(message, adObject) {
 export function getNativeTargeting(bid, bidReq) {
   let keyValues = {};
 
-  Object.keys(bid['native']).forEach(asset => {
-    const key = CONSTANTS.NATIVE_KEYS[asset];
-    let value = getAssetValue(bid['native'][asset]);
-
-    const sendPlaceholder = deepAccess(
-      bidReq,
-      `mediaTypes.native.${asset}.sendId`
-    );
-
-    if (sendPlaceholder) {
-      const placeholder = `${key}:${bid.adId}`;
-      value = placeholder;
-    }
+  if (deepAccess(bidReq, 'nativeParams.rendererUrl')) {
+    bid['native']['rendererUrl'] = getAssetValue(bidReq.nativeParams['rendererUrl']);
+  } else if (deepAccess(bidReq, 'nativeParams.adTemplate')) {
+    bid['native']['adTemplate'] = getAssetValue(bidReq.nativeParams['adTemplate']);
+  }
 
-    if (key && value) {
-      keyValues[key] = value;
+  Object.keys(bid['native']).forEach(asset => {
+    if (asset !== 'adTemplate') {
```

Comment:
I believe it suppose be 
```
if (asset !== 'adTemplate' && asset !== 'rendererUrl') {
```

Reply:
It was my understanding that rendererUrl can be sent as a KV in the GAM call; whereas adTemplate will never be sent. Is this not the case?

Reply:
From publisher point of view, I don't see the need to pass rendererUrl as a KV in GAM call. @jaiminpanchal27 , rendererUrl was originated from your team, what do you think about this?

Reply:
I looked into the user case in detail, and realize rendererUrl could pass as key/value. In case when different ad unit configures different rendererUrl, but share same creatives inside GAM.